src/epics/mutate.js

```diff
@@ -0,0 +1,91 @@
+import Rx from 'rxjs';
```

Comment:
mainly because the more I've learned about systems solving similar problems, the more I've come across that terminology - Relay, Falcor, and 'JSON-pure' all ignore POST/PATCH/DELETE distinctions and instead refer to them as 'mutations' - they are the HTTP requests that _change_ data, unlike GETs, which in theory should always be idempotent.